### PR TITLE
Update docs with nvme capabilities

### PR DESCRIPTION
**Pull request by artek-koltun**

### CODE REVIEW

File: build/storage/recipes/nvme/scale-out.md [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+Send from your `cmd-sender`
+```
+$ for ((i=0; i < "64"; i++)) ; do \
+	attach_volume <ipu_storage_container_platform_ip> "${devs[$i]}" "${ramdrives[$i]}" \
+    nqn.2016-06.io.spdk:cnode0 <storage_target_platform_ip> 4420 ; \
+done
+```
+
+
+8. Validate number of attached namespaces to the NVMe device.
```
Comment (intelfisz) [inline build/storage/recipes/nvme/scale-out.md:86]: namespaces attached to

File: build/storage/recipes/nvme/scale-out.md [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+```
+$ lsblk | grep -c nvme0n*
+```
+The Expected output is
+```
+32
+```
+
+
+12. Detach all namespaces and devices.
```
Comment (intelfisz) [inline build/storage/recipes/nvme/scale-out.md:127]: and delete all devices

File: build/storage/recipes/nvme/hot-plug.md [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+
+1. Perform all steps described in [environment setup](../environment_setup.md)
+
+
+2. Make sure there is no NVMe device/volumes attached.
+Run in [vm console](../environment_setup.md#vm-console)
+```
+$ lsblk; echo""; ls /dev/nvme*
+```
+The Expected output is
```
Comment (intelfisz) [inline build/storage/recipes/nvme/hot-plug.md:22]: expected (lower case)

File: build/storage/recipes/nvme/hot-plug.md [hunk 4]
```diff
@@ -0,0 +1,146 @@
+# Hot-plug recipe
+
+This recipe describes how to perform NVMe hot-plug to a running KVM host.
+The NVMe device namespaces are backed up by an ideal targets on
+`storage-target-platform` machine exposed over NVMe/TCP.
+
+For this recipe, two physical machines are required.
+They are referred as `storage-target-platform` and `ipu-storage-container-platform`.
```
Comment (intelfisz) [inline build/storage/recipes/nvme/hot-plug.md:8]: correct all occurrences to "referred to as"

File: build/storage/recipes/nvme/hot-plug.md [hunk 5]
```diff
@@ -0,0 +1,146 @@
+# Hot-plug recipe
+
+This recipe describes how to perform NVMe hot-plug to a running KVM host.
+The NVMe device namespaces are backed up by an ideal targets on
```
Comment (intelfisz) [inline build/storage/recipes/nvme/hot-plug.md:4]: stripe 'an' for plural form

**Review by intelfisz — CHANGES_REQUESTED** [review]
Correct nvme/NVMe to be used in a homogenous way - I prefer NVMe when it is used in text like 'NVMe device'.
Also consider changing 'vm' to 'VM' according to the argument above.

File: build/storage/recipes/nvme/scale-out.md (hunk 1 above)
Comment (artek-koltun) [reply build/storage/recipes/nvme/scale-out.md]: done

File: build/storage/recipes/nvme/scale-out.md (hunk 2 above)
Comment (artek-koltun) [reply build/storage/recipes/nvme/scale-out.md]: corrected

File: build/storage/recipes/nvme/hot-plug.md (hunk 3 above)
Comment (artek-koltun) [reply build/storage/recipes/nvme/hot-plug.md]: changed

File: build/storage/recipes/nvme/hot-plug.md (hunk 4 above)
Comment (artek-koltun) [reply build/storage/recipes/nvme/hot-plug.md]: adjusted

File: build/storage/recipes/nvme/hot-plug.md (hunk 5 above)
Comment (artek-koltun) [reply build/storage/recipes/nvme/hot-plug.md]: removed

**Review by intelfisz — APPROVED** [review]

**Pull request merged**